.github/workflows/test-build-windows.yml

```diff
@@ -0,0 +1,45 @@
+name: Test Builds on Windows
```

Comment:
Thanks a lot for add CI here! For consistency I guess we should rename the other workflow, so something like?  
`Test and package builds on Linux`
`Test builds on Windows`

I guess the tests succeed on Windows? Would be cool to add this as a step in the Windows CI as well.

Reply:
> Cool to see that CI passes. Could you try to split building and testing into two steps for better readability of the CI logs? I also wouldn't mind to define `_CRT_SECURE_NO_WARNINGS` to make msvc slightly less noisy.

I did the split. I suppressed the CRT secure warnings, NvAPI deprecation warnings, and fixed the spam about GCC pragma.

> Do you really need to copy `nvapi64.dll` into the system folder? When running the test suite with `[@unit-tests]` it doesn't run the `@system` tests which rely on `nvapi64.dll`, so as far as I remember, it should be ok without it. Alternatively just extending the `PATH` would be an alternative. But no strong feelings to keep the copy action.

I removed the System32 copy for now.

> And just to be sure, CI also fails when a test fails (thus the return code isn't swallowed somewhere in powershell)?

I checked that it's not swallowed with the build below. I'm not sure if we fail fast (if build errors) but it will end up failing anyway due to missing executable.

https://github.com/ishitatsuyuki/dxvk-nvapi/actions/runs/7585231717/job/20660611025

Reply:
FWIW we're still at 31 warnings, which is still quite a lot. But I'm not inclined to introduce more suppresses because all the warnings are kinda legit, e.g. the integer cast warnings.

Reply:
Yeah, perfectly. And I fully agree to not silence any more warnings.